sycamore/docset.py

```diff
@@ -218,6 +219,35 @@ def spread_properties(self, props: list[str], **resource_args) -> "DocSet":
         plan = SpreadProperties(self.plan, props, **resource_args)
         return DocSet(self.context, plan)
 
+    def augment_text(self, augmentor: TextAugmentor, **resource_args) -> "DocSet":
+        """
+        Augments text_representation with external information.
```

Comment:
You might need to add a directive in the docs TOC to make sure this gets included.

Reply:
ack